lib/rules/max-line-length/README.md

```diff
@@ -124,3 +124,15 @@ The following pattern is *not* considered a violation:
 ```css
 @import "../../../../another/css/or/scss/file/or/something.css";
 ```
+
+Given the following, with a maximum length of `20`.
+
+```js
+["/https?:\/\/[0-9,a-z]*.*"]
```

Comment:
Looks like you've [forgotten](https://github.com/stylelint/stylelint/pull/2636/files#diff-eaf982a53e894d176d66da1033c4de50R241) an ending `/`.